src/setupTests.js

```diff
@@ -5,3 +5,4 @@
 import '@testing-library/jest-dom/extend-expect';
 import 'jest-prop-type-error';
 import 'jest-canvas-mock';
+window.URL.createObjectURL = function() {};
```

Comment:
This was the root cause of plotly breaking the test suite. 

The fix was found here: https://stackoverflow.com/questions/57943736/how-to-fix-window-url-createobjecturl-is-not-a-function-when-testing-mapbox-gl?rq=1